string maneuver
string object
string target
string dilated_target
string forbidden
string maneuver_mesh
string new_target
string new_object
---
int8 have_maneuver
int8 mesh_object_loaded
int8 mesh_target_loaded
int8 mesh_forbidden_loaded
int8 operation_done
float64[] parameters

